openlcs/openlcs/settings.py

```diff
@@ -159,6 +160,21 @@
 CELERY_RESULT_SERIALIZER = 'json'
 CELERY_RESULT_BACKEND = 'db+postgresql://{USER}:{PASSWORD}@{HOST}/{NAME}'.format(**DATABASES.get('default'))    # noqa
 CELERY_TASK_TRACK_STARTED = True
+CELERY_BEAT_SCHEDULER = "django_celery_beat.schedulers:DatabaseScheduler"
+CELERY_BEAT_SCHEDULE = {
+    # TODO add periodic task conf
+    # refer to https://docs.celeryq.dev/en/v5.2.7/userguide/periodic-tasks.html#beat-custom-schedulers
```

Comment:
Is this link correct?

Reply:
Thanks for reminding. I will change the link.